### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nextcloud/cypress",
-  "version": "1.0.0",
```
Comment (skjnldsv) [inline package.json:3]: Let's release 1.0.0 first? xD

Comment (skjnldsv) [reply package.json:3]: Also, maybe as beta.0 ?
```suggestion
  "version": "1.0.0-beta.0",
```

Comment (artonge) [reply package.json:3]: Already did but without build files :see_no_evil:

Comment (artonge) [reply package.json:3]: Need a new tag now

Comment (skjnldsv) [reply package.json:3]: Why is this merged?

**Pull request merged**